MSAL/test/integration/native_auth/end_to_end/reset_password/MSALNativeAuthResetPasswordEndToEndTests.swift

```diff
@@ -24,9 +24,12 @@
 
 import Foundation
 import XCTest
+import MSAL
 
 final class MSALNativeAuthResetPasswordEndToEndTests: MSALNativeAuthEndToEndBaseTestCase {
     // Hero Scenario 3.1.1. SSPR – without automatic sign in
+    let numberOfRetries = 3
```

Comment:
retries of what? Of the entire test or just the code sent?  Can this attribute be part of the MSALNativeAuthEndToEndBaseTestCase class instead?

Reply:
Changed the name to be more clear. It retries retrieving and submitting the code. also made it private.